decidim-system/spec/system/organizations_spec.rb

```diff
@@ -88,6 +88,29 @@
       end
     end
 
+    describe "edits organization name" do
```

Comment:
The title says "edits organization name" but then what we're doing in creating a new organization. By looking to the other examples/contexts in this file, I think this should be a context inside "creating an organization" called " with an invalid organization admin name" with an example "does not create an organization" to be consistent. This should be the final result:

```
    creating an organization
      has some fields filled by default
      creates a new organization
      behaves like form hiding advanced settings
        hides advanced settings
      with an invalid organization admin name
        does not create an organization
      with invalid data
        does not create an organization
      without the secret key defined
        does not create an organization
```

Reply:
It should be read like a phrase "creating an organization with an invalid organization admin name does not create an organization"  (sometimes this is tricky or almost impossible, but the aspiration at least should be that)